### ISSUE TITLE
Call "inline call to getter" does not work correctly with @Getter

### ISSUE THREAD

**Issue by pmiddend** [mplushnikov/lombok-intellij-plugin#155]
Instead of replacing the getter with a direct call of the instance variable a new line is created with an uninitialized local variable of the instance variables class.

Example:

    @Getter
    static String foo = "foo";
    public static void main(final String[] args) {
        final String bar = getFoo();
    }

becomes

    @Getter
    static String foo = "foo";
    public static void main(final String[] args) {
        final String result;
        final String bar = result;
    }

**Comment by mplushnikov** [comment]
I cannot reproduce it on the current version of Intellij (2016.1) and Plugin (0.10).
I only get message "Cannot perform refactoring"
![image](https://cloud.githubusercontent.com/assets/1204509/14061165/cde107b8-f379-11e5-92a7-667b9b47e37f.png)

Can you describe your steps to reproduce the issue?

**Comment by pmiddend** [comment]
Thanks for trying, I will get back to you on that on tuesday.

**Comment by noxxs** [comment]
These are the steps that I am taking:

Before intent action:
![before](https://cloud.githubusercontent.com/assets/12800146/14098752/647658f2-f57f-11e5-95c5-4df6853b2667.png)

intent dialog:
![action](https://cloud.githubusercontent.com/assets/12800146/14098808/ec2d1650-f57f-11e5-97ab-33b40f71a6bc.png)

Result of intent action:
![after](https://cloud.githubusercontent.com/assets/12800146/14098751/644dd9e0-f57f-11e5-9ab7-fbc8a8c2f1be.PNG)

Maven example project:
[getterrefactoring.zip](https://github.com/mplushnikov/lombok-intellij-plugin/files/192899/getterrefactoring.zip)

**Comment by AlexejK** [comment]
@mplushnikov, I did a very quick test on this and found some interesting behaviour.

Provided reference project doesn't have default annotation processing enabled, but Maven Annotation processing is enabled. Upon removal of all `*.iml` files as well as `.idea` and re-import of the project with Default annotation processing enabled - the refactoring intent is not even shown.

By creating a project from scratch I can't trigger this behaviour either. I have a feeling this has something to do with old project file generation that does not take into account annotation processing.

**Comment by Janmm14** [comment]
There is an option for default annotation processing in the settings in the compiler section.

**Comment by AlexejK** [comment]
Yes, the point is, it was not enabled and even simply enabling it does not seem to help, while the full re-import does help. and removes the intent as well. Sample project has "maven annotation processing" enabled while default one was disabled. Both enabled did not do much either.

**Comment by Janmm14** [comment]
Okay.
